packages/react/src/Form/InputNumber/Number.tsx

```diff
@@ -0,0 +1,33 @@
+import { ComponentPropsWithRef, forwardRef, useId } from "react";
+import { getComponentClassName } from "../core";
+import "@axa-fr/design-system-css/dist/Form/InputText/InputText.agent.scss";
+
+type Props = Omit<ComponentPropsWithRef<"input">, "required" | "type"> & {
+  classModifier?: string;
+};
+
+const Number = forwardRef<HTMLInputElement, Props>(
+  ({ id, className, classModifier, ...otherProps }, inputRef) => {
+    const inputUseId = useId();
+    const inputId = id ?? inputUseId;
```

Comment:
```suggestion
    const inputId = id || inputUseId;
```
Grâce à ça, si `id` est une string vide on va utiliser `inputUseId`.

Reply:
un hook sera créé pour gérer cela : #79